src/engraving/infrastructure/shape.h

```diff
@@ -160,8 +160,8 @@ class Shape
     double leftMostEdgeAtHeight(double yAbove, double yBelow) const;
 
     bool contains(const PointF&) const;
-    bool intersects(const RectF& rr) const;
-    bool intersects(const Shape&) const;
+    bool intersects(const RectF& rr, double horClearance = 0.0, double vertClearance = 0.0) const;
```

Comment:
I don't really like these arguments... It seems like they're just syntactic sugar, but they actually make the code harder to understand... IMO, it's better to call adjust() where we need it and pass only the rect to these methods